x-pack/legacy/plugins/siem/public/components/timeline/body/renderers/get_row_renderer.test.tsx

```diff
@@ -72,4 +77,80 @@ describe('get_column_renderer', () => {
       'some child 4ETEXPLOITNETGEARWNR2000v5 hidden_lang_avi Stack Overflow (CVE-2016-10174)Source192.168.0.3:53Destination192.168.0.3:6343'
     );
   });
+
+  test('should render a suricata row data if event.category is network_traffic', () => {
+    suricata.event = { ...suricata.event, ...{ category: ['network_traffic'] } };
+    const rowRenderer = getRowRenderer(suricata, rowRenderers);
+    const row = rowRenderer.renderRow({
+      width: 100,
```

Comment:
This uses the older widths before the change over to the widths that are context properties. So I have to add these here in `7.3`.